### DISCUSSION TITLE
Subscription Management

### DISCUSSION THREAD

**Discussion by techch01** [Azure/ipam#25] [General]
Hello,

As I want to manage a scope of subscriptions, how can I exclude subscriptions and then see the list of excluded subscriptions ?

Thanks.
Best regards,

**Comment by DCMattyG** [comment]
Hi @techch01, thanks for reaching out!

In the IPAM, you can navigate to the "Subscriptions" section under "Admin". Here, you'll find a list of all the subscriptions and you can click on the ones you'd like to exclude (this turns them red). When done, click the "save" button in the upper right.

Screenshot:

![image](https://user-images.githubusercontent.com/12955732/191771151-1665633f-1451-4939-8ceb-fe92f38189f9.png)

Hope that helps, please let us know if you have any problems at all!

**Reply by techch01** [reply]
Hi @DCMattyG 

It is OK. Thanks for your return.

**Comment by techch01** [comment]
Hi @DCMattyG,

When a subscription is excluded, that is mean that its Vnets/subnets/Endpoints are not discoverd by the IPAM, isn't it ?

Thanks in advance?

**Reply by DCMattyG** [reply]
100% Correct. Everything in that subscription is ignored.

**Reply by techch01** [reply]
Hello @DCMattyG 

First, happy new year !

I get a empty screen when I want to get "Subscriptions" from "Admin" tab.
The GET Invoke-RestMethod return is OK.

![image](https://user-images.githubusercontent.com/35137280/211760412-12db783f-866c-4bdc-aa23-fee7adb5b389.png)

Before it works fine, but now it isn't.  

Any idea from your side ?
Thanks in advance.


Best regards,
Tariq ECHCHAITAMI

**Reply by DCMattyG** [reply]
Hey @techch01, it's strange because in the screenshot above, it doesn't appear that the browser even navigated to the Subscriptions page (I can't see the address bar). Could you perhaps try in another browser just to verify this behavior is consistent?

**Reply by techch01** [reply]
Hi @DCMattyG,

Please to find below the screen : 

![image](https://user-images.githubusercontent.com/35137280/211873015-0f67dffe-13ff-45d7-acbd-d44dab250fc1.png)

**I have changed the brower and I have the same result.**

Please to find below the Network trace of URL "https://**name_webapp**.azurewebsites.net/admin/subscriptions"

![image](https://user-images.githubusercontent.com/35137280/211874738-e1291f01-bd09-4d50-b529-356566999e9f.png)


![image](https://user-images.githubusercontent.com/35137280/211874927-6950fd60-9edf-498e-ae26-be913ea3156f.png)


Thanks in advance.

Best regards,
Tariq ECHCHAITAM

**Reply by DCMattyG** [reply]
This is indeed strange. It's getting back a 200 OK, but not showing the table (for some reason).

Can you try restarting your App Service in Azure? I just tested this with one of my deployments and I'm not seeing the same behavior. That certainly doesn't mean there isn't something going on, but lets try a simple restart first.

**Reply by techch01** [reply]
Hi @DCMattyG 

 I tried a restart and I have the same behavior !

Best regards,
Tariq ECHCHAITAM

**Reply by DCMattyG** [reply]
Hey @techch01, lets setup some time to look at this together. I can't seem to re-create this behavior, so it's difficult for me to figure out what's going on.

Would you mind reaching out to me at Matthew.Garrett@microsoft.com so we can setup a 1:1 to figure out what's wrong?
